client/verta/tests/conftest.py

```diff
@@ -514,9 +514,9 @@ def organization(client, created_entities):
 
 @pytest.fixture
 def requirements_file():
-    with tempfile.NamedTemporaryFile("w+") as tempf:
+    with tempfile.NamedTemporaryFile("w+", delete=False) as tempf:
```

Comment:
Suggestion:

- `delete=False` is 👌, though a bit odd to me
- line 526: move `yield tempf` out of the `with:` block (unindent one level)'
- after the `yield`, add an `os.remove(tempf.name)`

With this, we can take out some of the `.close()`s and `os.remove(tempf.name)`s added to tests in this PR